src/utils/actions.js

```diff
@@ -140,11 +141,12 @@ const actions = [
     group: "selection",
     getActionCall:
       ({ app, qStateName, field, value }) =>
-      async () => {
-        if (field && value) {
+      async (overrideValue) => {
+        const useValue = overrideValue || value;
```

Comment:
yeah as Christofer mentioned, this could fail if overrideValue is `0` or `""`
```suggestion
        const useValue = overrideValue ?? value;
```